common/src/main/java/com/netflix/conductor/common/run/TaskNotification.java

```diff
@@ -10,39 +10,46 @@
  * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
  * specific language governing permissions and limitations under the License.
  */
-package com.netflix.conductor.contribs.publisher;
+package com.netflix.conductor.common.run;
```

Comment:
TaskNotification and WorkflowNotification are specific to Publish module. I think they should remain in contribs project. Any specific reasons for this change?